gradle-plugin/src/main/kotlin/com/google/devtools/ksp/gradle/KspSubplugin.kt

```diff
@@ -220,10 +222,14 @@ class KspGradleSubplugin @Inject internal constructor(private val registry: Tool
             kspTask.destination = destinationDir
             kspTask.outputs.dirs(kotlinOutputDir, javaOutputDir, classOutputDir, resourceOutputDir)
             kspTask.blockOtherCompilerPlugins = kspExtension.blockOtherCompilerPlugins
+
+            // depends on the processor; if the processor changes, it needs to be reprocessed.
+            val processorClasspath = project.configurations.maybeCreate("ProcessorClasspath")
```

Comment:
For Android projects, there will be multiple KSP tasks, so this name will not be unique. Consider using "${taskName}ProcessorsClasspath".

Reply:
Done. Thanks!